src/App.js

```diff
@@ -7,7 +7,11 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Edit
+          {' '}
```

Comment:
- [ ] The `App.js`(main component) should have the imported `Calculator.js` component